Revision of Applicability draft (#117)
quicwg/ops-drafts: draft-ietf-quic-applicability.md

applications can assume that any NAT binding or other state entry will be
expired after just thirty seconds of inactivity.

By using a Connection ID, QUIC is designed to be robust to NAT address
rebinding after a timeout. However, some QUIC connections may not be robust to
rebinding because the routing infrastructure (in particular, load balancers)

> uses the address/port fourtuple to direct traffic. Furthermore, middleboxes

Reviewer: ```suggestion
uses the address/port four-tuple to direct traffic. Furthermore, middleboxes
```

with functions other than address translation may still affect the path. In
particular, firewalls will often not admit server traffic for which it has not
kept state for corresponding packets from the client.

A QUIC application has three strategies to deal with this issue:

- Ignore it, if the application-layer protocol consists only of interactions

>   with no or very short idle periods, or the protocol's resistance to NAT

Reviewer: ```suggestion
  with no or very short idle periods, or the application's resistance 
  to NAT
```

Reviewer 2: Not sure I understand this point. Isn't "simply reconnect" on application layer point three below?

Author: what this bullet means is the QUIC NAT rebinding mechanism is working fine -- there is no 4-tuple router breaking everything. So this suggested change breaks the meaning, though perhaps it's a better or equally relevant point.

stream is mapped into frames or how those frames are bundled into packets.
By default, QUIC will try to maximally pack packets with one or more stream
data frames to minimize bandwidth consumption and computational costs (see

> section 13 of {{!QUIC}}). If there is not enough data available to fill a packet,

Reviewer: ```suggestion
section 13 of {{!QUIC}}). If there is not enough data available to fill a 
packet,
```

Reviewer: (80-char limit, we can rewrap after submit)
